File: DataBindingSample/app/src/main/java/com/example/databindingsample/MainActivity.java
package com.example.databindingsample;

import android.databinding.DataBindingUtil;
import android.databinding.ObservableArrayMap;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;

import com.example.databindingsample.databinding.ActivityMainBinding;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        ActivityMainBinding binding = DataBindingUtil.setContentView(this, R.layout.activity_main);

        ObservableArrayMap<String, Object> user = new ObservableArrayMap<>();
        user.put("firstName", "Google");
        user.put("lastName", "Inc.");
        user.put("age", 17);

//        binding.setUser(user);
    }
}

File: WorkManager/app/src/main/java/com/example/workmanager/MainActivity.kt
package com.example.workmanager

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.work.OneTimeWorkRequestBuilder
import androidx.work.WorkManager

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val githubWorkManager = OneTimeWorkRequestBuilder<GitHubWork>()
                .build()
        WorkManager.getInstance(applicationContext).enqueue(githubWorkManager)
    }
}